mass/usask/getimages.py

```
@@ -15,33 +15,33 @@
     with open(file_path, "r") as file:
         data = json.load(file)
     return data
 
-
 def extract_images_from_url(url):
     # Print the URL being processed
-    printe.output(f"\t Processing URL: {url}")
+    logger.info(f"\t Processing URL: {url}")
 
     # Send a GET request to the URL and get the response
     response = requests.get(url)
```

Ruff (0.15.0)

[error] Probable use of `requests` call without timeout

(S113)